app/move/controllers/create/personal-details.test.js

```diff
@@ -150,55 +185,31 @@ describe('Move controllers', function() {
         }
       })
 
-      context('when person does not exist', function() {
-        context('when save is successful', function() {
-          beforeEach(async function() {
-            sinon.stub(personService, 'create').resolves(personMock)
-            sinon.spy(personService, 'update')
-            req.sessionModel.get.withArgs('person').returns(undefined)
-            await controller.saveValues(req, {}, nextSpy)
-          })
-
-          it('should create a person via the personService', function() {
-            expect(personService.create).to.be.calledOnce
-            expect(personService.update).to.not.be.called
-          })
-
-          it('should set person response on form values', function() {
-            expect(req.form.values).to.have.property('person')
-            expect(req.form.values.person).to.deep.equal(personMock)
-          })
+      context('when save is successful', function() {
+        beforeEach(async function() {
+          sinon.stub(personService, 'create').resolves(personMock)
```

Comment:
I think it would be safer to stub `this.savePerson` in this unit test otherwise you're testing that function of `savePerson` twice.

And then you can just check that `this.savePerson` is being called with the correct values here.